ArduPlane/ArduPlane.cpp

```diff
@@ -1021,10 +1021,10 @@ void Plane::update_soaring() {
         if (soaring_controller.check_init_thermal_criteria()) {
             // start by circling around current location
             soaring_controller.set_wp(next_WP_loc);
-            soaring_controller.update_thermalling(g.loiter_radius); // update estimate
+            soaring_controller.update_thermalling(g.thermalling_radius); // update estimate
         } else {
             // Update thermal estimate and check for switch back to AUTO
-            soaring_controller.update_thermalling(g.loiter_radius);  // Update estimate
+            soaring_controller.update_thermalling(g.thermalling_radius);  // Update estimate
```

Comment:
Actually, I just realized that currently the code doesn't make use of both thermalling radius and loiter radius -- just having the "global" loiter radius (in aparm) is enough. So, I removed the "local" thermalling radius.